srcpkgs/mdcat/template

```diff
@@ -22,14 +22,18 @@ case "$XBPS_TARGET_MACHINE" in
 esac
 
 post_install() {
-	bash_completion=$(find ${wrksrc}/target -name mdcat.bash -print -quit)
-	vcompletion ${bash_completion} bash
-	zsh_completion=$(find ${wrksrc}/target -name _mdcat -print -quit)
-	vcompletion ${zsh_completion} zsh
-	fish_completion=$(find ${wrksrc}/target -name mdcat.fish -print -quit)
-	vcompletion ${fish_completion} fish
+	for command in mdcat mdless; do
+		bash_completion=$(find ${wrksrc}/target -name ${command}.bash -print -quit)
+		vcompletion ${bash_completion} bash ${command}
+		zsh_completion=$(find ${wrksrc}/target -name _${command} -print -quit)
+		vcompletion ${zsh_completion} zsh ${command}
+		fish_completion=$(find ${wrksrc}/target -name ${command}.fish -print -quit)
+		vcompletion ${fish_completion} fish ${command}
+	done
 
 	man_page=$(find ${wrksrc}/target -name mdcat.1 -print -quit)
 	vman ${man_page}
+
 	ln -sf /usr/bin/mdcat ${DESTDIR}/usr/bin/mdless
+	ln -sf mdcat.1 ${DESTDIR}/usr/share/man/man1/mdless.1
```

Comment:
i don't see any other distro doing this, and the help output for mdless says `See 'man 1 mdcat' for more information.`

Reply:
Still, here's the synopsis for the manpage:
```
       mdcat [OPTIONS] [FILE]...

       mdless [OPTIONS] [FILE]...
```

So I think it makes sense.

Reply:
Nope, I don't think it make sense, its `.TH` is `MDCAT`, its `.SH "NAME"`  is `mdcat`, upstream should change like this:
```diff
30c30
< mdcat \- render CommonMark Markdown to text terminals
---
> mdcat, mdless \- render CommonMark Markdown to text terminals
```

Reply:
I understand, I removed the manpage symlink for now.